jin/(6월 1주차 과제)/10952.py

```diff
@@ -1,5 +1,5 @@
 while True:
-    a,b = majp(int, input().split())
+    a,b = map(int, input().split())
     if a+b !=0:
```

Comment:
```suggestion
    if a != 0 and b !=0:
```
문제 조건이 자연수가 아닌 정수이기 때문에 합이 아닌 각각의 값으로 검사하는 게 더 정확할 것 같아요!